linode/manager
# upcoming: [M3-7693] - Rename `Akamai Global Load Balancer` to `Akamai Cloud Load Balancer`

## Description 📝

- Goes all in on the `Akamai Global Load Balancer` to `Akamai Cloud Load Balancer` rename ✏️
- Yes, I even created new feature flags 🚩 and deprecated the old ones. This probably is very extra of me, but I figured now would just be a good time to get it over with

> [!important]
> The only thing that shouldn't change as part of this PR is API endpoint URLs and Docs links URLs to prevent breaking dev for internal testers. We will do a separate PR for that and coordinate with the API team

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-01-26 at 12 07 09 PM](https://github.com/linode/manager/assets/115251059/4d8abfd7-18dc-40d7-bd33-233267328d2d) | ![Screenshot 2024-01-26 at 12 06 48 PM](https://github.com/linode/manager/assets/115251059/cfca7460-e174-44e3-9d06-bbfd0773d0bf) |


## How to test 🧪

### Prerequisites
- You can test with the MSW **or** use `dev-test-aglb` with the dev API

### Verification steps 
- Verify you see  to `Akamai Cloud Load Balancer` instead of `Akamai Global Load Balancer` in the UI
- Look at the diff and check my find-and-replace work

## As an Author I have considered 🤔
